load.php

```diff
@@ -25,6 +25,11 @@
 define( 'PERFLAB_MODULES_SETTING', 'perflab_modules_settings' );
 define( 'PERFLAB_MODULES_SCREEN', 'perflab-modules' );
 
+// Load Composer dependencies if applicable.
+if ( file_exists( PERFLAB_PLUGIN_DIR_PATH . '/vendor/autoload.php' ) ) {
+	require_once PERFLAB_PLUGIN_DIR_PATH . '/vendor/autoload.php';
+}
```

Comment:
While Composer is generally great, using it in a WordPress plugin comes with additional quirks and interoperability problems that I would prefer to avoid unless truly beneficial. Let's `require_once` any class files instead, as there shouldn't be any meaningful performance benefit from autoloading those few files anyway.